blocks/components/editable/index.js

```diff
@@ -1,3 +1,74 @@
-export default function Editable( { value } ) {
-	return <p>{ value }</p>;
+export default class Editable extends wp.element.Component {
+	constructor() {
+		super( ...arguments );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.bindNode = this.bindNode.bind( this );
+	}
+
+	componentDidMount() {
+		this.initialize();
+	}
+
+	initialize() {
+		const config = {
```

Comment:
To recreate default `contenteditable` behaviour, we should probably add `browser_spellcheck: true`.